### PR TITLE
added current location to data sent to api

### PR DESCRIPTION
**Pull request by inteljudajocu**
#### Description:

Adding the current location to the data sent to the API receiving the data from the form

#### Linked issues:

[Related ticket](https://trello.com/c/YlLMFYyU/96-as-a-user-i-want-to-share-my-latitude-and-length-when-the-form-is-completed-so-that-health-authorities-can-know-my-location)

#### Screenshots:

NA

#### How to test:

Complete the form and [This is the endpoint](https://webapps.mepyd.gob.do:443/contact_tracing/api/Form)

### CODE REVIEW

File: app/views/DR/ReportScreen/sections/SetpAdress.js [hunk 1]
```diff
@@ -46,6 +48,23 @@ const StepAdress = ({ setCompleted }) => {
...
+    const granted = await PermissionsAndroid.check(
+      PermissionsAndroid.PERMISSIONS.ACCESS_FINE_LOCATION,
+    );
+    if (granted || Platform.OS === 'ios') {
+      Geolocation.getCurrentPosition(
+        ({ coords }) => {
+          setSelectedOption('latitude', coords.latitude.toString());
+          setSelectedOption('longitude', coords.longitude.toString());
+        },
+        () => {},
```
Comment (JoseFcoEstrella) [inline app/views/DR/ReportScreen/sections/SetpAdress.js]: Why this arrow function?

Comment (inteljudajocu) [reply app/views/DR/ReportScreen/sections/SetpAdress.js]: I miss it when cleaning. Will remove it now

Comment (inteljudajocu) [reply app/views/DR/ReportScreen/sections/SetpAdress.js]: I missed it when cleaning, will remove it now

**Review by JoseFcoEstrella — APPROVED** [review]
LGTM

File: app/views/DR/ReportScreen/sections/SetpAdress.js [hunk 2]
```diff
@@ -143,6 +161,7 @@ const StepAdress = ({ setCompleted }) => {
                   } else if (text === '') {
                     setSelectedOption('numberPersonLivesWith', '');
                   }
+                  getCurrentLocation();
```
Comment (adnanjt) [inline app/views/DR/ReportScreen/sections/SetpAdress.js:164]: Esto se envia para el formulara a las respuestas al final y se salva en el stater tambien por si se necesita en el otro screem ??

Comment (inteljudajocu) [reply app/views/DR/ReportScreen/sections/SetpAdress.js:164]: Se envia al global state directamente, si se necesita en otro screen solo habría que acceder al global state(a su latitud o longitud)

Comment (adnanjt) [reply app/views/DR/ReportScreen/sections/SetpAdress.js:164]: Nice this is done

**Review by adnanjt — APPROVED** [review]
LGTM

**Comment by adnanjt** [comment]
Ready to merge

**Pull request merged**